### ISSUE TITLE
Fresh Laravel installation issue

### ISSUE THREAD

**Issue by mohamedary** [joelbutcher/laravel-facebook-graph#20]
I tried to install this package with a very fresh installation of Laravel `10.16.1`

and ran into the following errors:
<img width="1403" alt="image" src="https://github.com/joelbutcher/laravel-facebook-graph/assets/17085444/6a535d81-8158-4fbf-9a74-6312f85d9530">

Here is my composer.json file:

```json
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The skeleton application for the Laravel framework.",
    "keywords": ["laravel", "framework"],
    "license": "MIT",
    "require": {
        "php": "^8.1",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^10.10",
        "laravel/sanctum": "^3.2",
        "laravel/tinker": "^2.8"
    },
    "require-dev": {
        "fakerphp/faker": "^1.9.1",
        "laravel/pint": "^1.0",
        "laravel/sail": "^1.18",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^7.0",
        "phpunit/phpunit": "^10.1",
        "spatie/laravel-ignition": "^2.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true,
        "allow-plugins": {
            "pestphp/pest-plugin": true,
            "php-http/discovery": true
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true
}
```

**Comment by mohamedary** [comment]
@joelbutcher

**Comment by joelbutcher** [comment]
[v1.2.3](https://github.com/joelbutcher/laravel-facebook-graph/releases/tag/v1.2.3) should support Laravel 10

**Comment by joelbutcher** [comment]
> @joelbutcher 

I'm currently on holiday, so any time I'm giving to responding to raised issues is not done immediately @'ing me won't speed up the process.

I cannot replicate this issue with the latest release, so closing this issue.

**Comment by mohamedary** [comment]
@joelbutcher I obviosuly did know that you are on holiday. Also, I tried pinging you because I was unsure whether you are actively maintaining this repo or not. However, enjoy your holiday.

Fixed by adding `"joelbutcher/laravel-facebook-graph": "^1.2"` to `composer.json` and running `composer update`. Looks like I needed to update dependencies first which is weird because I used create-project then installed the package right after.

Anyway, 
Peace out :)
